src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,7 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "countries")
 public class Country implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "country_id")
     private Long id;
+    @Column(name = "country_name")
```

Comment:
```suggestion
```
Don't need to specify country, because name column already related to the country table
Check all places